src/components/Accounts/ChargeFeeFine/UserInfo.js

```diff
@@ -20,7 +20,7 @@ const UserInfo = (props) => {
         </Col>
         <Col className={css.userInfoLink}>
           <FormattedMessage id="ui-users.charge.barcode" />
-          {': '}
+          :
```

Comment:
Heh, freaking ESLint, man. Are you confident this will show up as `Barcode: 12345` with a space after the colon? Previously, this looked like
```
<FormattedMessage id="ui-users.charge.barcode" />
<Link to={`/users/view/${user.id}`}>
  {user.barcode}
</Link>
```
and the translation string was variously coded with and without the trailing space, in fact with and without the colon at all 🤦 , and I just couldn't handle seeing `Barcode:1234` any longer. The horror. The HORROR!